src/components/Settings/index.tsx

```diff
@@ -28,8 +28,8 @@ const StyledMenuIconContainer = styled.div`
   justify-content: center;
   align-items: center;
   padding: 0px 8px;
-  height: 29px;
-  width: 29px;
+  height: 32px;
+  width: 32px;
```

Comment:
This fixes the Setting icon to be 32px

![image](https://user-images.githubusercontent.com/1926216/144941028-78cebf0b-2832-4585-9e37-de20120ca6ec.png)